src/main/resources/aspects/FeaturedContent/i18n/messages.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<catalogue xml:lang="en" xmlns:i18n="http://apache.org/cocoon/i18n/2.1" xmlns="http://apache.org/cocoon/i18n/2.1">
+
+    <!--
+      The format used by all keys is as follows
+
+      xmlui.<Aspect>.<Java Class>.<name>
+
+      There are a few exceptions to this nameing format,
+      1) Some general keys are in the xmlui.general namespace
+         because they are used very frequently.
+      2) Some general keys which are specific to a particular aspect
+         may be found at xmlui.<Aspect> without specifiying a
+         particular java class.
+      -->
+
+    <message key="xmlui.aspect.FeaturedContent.FeaturedContentForm.head">Featured Content</message>
+
+    <message key="xmlui.aspect.FeaturedContent.FeaturedContentForm.trail">Manage Featured Content(trail)</message>
```

Comment:
Please change this to remove the "(trail)", it will be confusing to our users when they test this functionality. Likewise for the other places in this file please.